src/browser/commands/assert-view/index.js

```diff
@@ -155,7 +155,23 @@ module.exports.default = browser => {
         testplaneCtx.assertViewResults.add({ stateName: state, refImg: refImg });
     };
 
-    session.addCommand("assertView", async function (state, selectors, opts = {}) {
+    session.addCommand("assertView", async function (state, selectorsOrOpts, opts = {}) {
+        let selectors;
+        if (selectorsOrOpts) {
+            if (typeof selectorsOrOpts === "string" || _.isArray(selectorsOrOpts)) {
+                selectors = selectorsOrOpts;
+            } else if (typeof selectorsOrOpts === "object") {
+                opts = selectorsOrOpts;
+            }
+        }
+        if (!selectors) {
+            opts = _.defaults(opts, {
+                allowViewportOverflow: true,
+                compositeImage: false,
+                captureElementFromTop: false,
+            });
+            selectors = "body";
+        }
         await Promise.map([].concat(selectors), async selector => {
```

Comment:
Should we wait `body` in case when user assert viewport?

Reply:
Not sure as body should probably always exist